src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -59,7 +59,7 @@ public class TestJsonCommand : PSCmdlet
         /// <returns>Return value is unreachable since we always rethrow.</returns>
         private static bool UnwrapException(Exception e)
         {
-            if (e is TargetInvocationException)
+            if (e.InnerException != null && e is TargetInvocationException)
             {
                 ExceptionDispatchInfo.Capture(e.InnerException).Throw();
             }
```

Comment:
Do we need to add an else?
cc @daxian-dbw

Reply:
The check `e.InnerException != null` is not needed. The inner exception of `TargetInvocationException` is always not null.

See https://docs.microsoft.com/en-us/dotnet/api/system.reflection.targetinvocationexception?view=netcore-3.1
> When created, the TargetInvocationException is passed a reference to the exception thrown by the method invoked through reflection. The InnerException property holds the underlying exception.

Reply:
It is annotated as nullable and compiler warns us
https://source.dot.net/#System.Private.CoreLib/TargetInvocationException.cs,12